tests/web/api/test_solidity_source_import.py

```diff
@@ -48,6 +93,6 @@ def test_importing_solidity_source_code(api_client, factories):
 
     assert response.status_code == status.HTTP_201_CREATED
     data = response.data
-    assert data['num_processed'] == 8
-    assert data['num_imported'] == 6
+    assert data['num_processed'] == 17
+    assert data['num_imported'] == 15
     assert data['num_duplicates'] == 2
```

Comment:
Is there a reason that you chose not to make any assertions about the `num_ignored`?  I seem to recall reviewing some code on the serializers that added that value to the returned data.

Reply:
The current [implementation](https://github.com/pipermerriam/ethereum-function-signature-registry/blob/master/func_sig_registry/registry/serializers.py#L74) does not return ignored elements. Should I implement that?

Reply:
Given that [it has been implemented on the other serializers](https://github.com/pipermerriam/ethereum-function-signature-registry/blob/aa2ff05a36017c7a1ab568ea9f4e51e244245f4f/func_sig_registry/registry/serializers.py#L93) I think it would be worth staying consistent so yes.